### ISSUE TITLE
Markdown support

### ISSUE THREAD

**Issue by osmsnbey** [jendrikseipp/rednotebook#560]
As a technical savy user i want to write my entries using markdown so that i can use postprocessing easier.

Postprocessing is for example converting whole journal entries (single or from x to y range) into pdf or html documents. 
Also using markdown (including extensions like mermaid, katex and so on) it should be possible to generate very powerful documentation.

**Comment by jendrikseipp** [comment]
I agree that it would be nice to have Markdown support. If you or anybody else would like to work on this, please get in touch.

**Comment by pdelrot-samea** [comment]
Hello,
Markdown would certainely be appreciated, for me, I switched to asciidoc format few months ago.
Asciidoc format support may be added too !
[https://asciidoc.org/](Asciidoc.org)
Pascal

**Comment by jagercode** [comment]
YES, please!

**Comment by biffster** [comment]
Add me as a +1 for Markdown support. txt2tags is too limiting, at least in the way my brain works.

**Comment by rogercreagh** [comment]
Markdown will be pretty essential for exchanging information through CalDav with other applications. I guess you will need to add at FMTTYPE parameter in the file for the metadata about what type of encoding is in use for the entry - text/markdown text/html etc.

**Comment by github2099** [comment]
+1 for markdown support, that would solve a lot of problems with YAML

**Comment by ederlf** [comment]
@jendrikseipp I would be keen on trying to add markdown support. Do you already have a vision about how you would like it to be integrated or is it still something to discuss?
